conf/application.conf

```diff
@@ -216,6 +210,7 @@ urls {
   cdsCheckStatus = "https://www.tax.service.gov.uk/customs/register-for-cds/are-you-based-in-uk"
   feedbackFrontend = "http://locahost:9514/feedback/cds-file-upload-frontend"
   nationalClearingHubLink = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/national-clearance-hub"
+  emailFrontendUrl = "http://localhost:9898/manage-email-cds/service/cds-file-upload"
```

Comment:
Please create a PR for `app-config-base` to set this value for all environments. From what I remember it should be just `/manage-email-cds/service/cds-file-upload` there.